app/vmalert/config/testdata/rules/group-interval-offset-explicit-bad.rules

```diff
@@ -0,0 +1,14 @@
+groups:
+  - name: EvalOffsetGroup
+    interval: 300
+    interval_offset: 360
```

Comment:
How do you expect such config should work:
```
    interval: 300
    interval_offset: 360
```
When exactly this rule will be evaluated?